app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less

```diff
@@ -105,11 +105,17 @@
 //
 
 .admin__collapsible-block-wrapper {
+    .admin__collapsible-title[aria-expanded="true"]{
+        &:before {
+            content: @icon-expand-close__content;
+        }
+    } 
+    
     .__collapsible-block-wrapper-pattern();
     .admin__collapsible-title {
         .__collapsible-title-pattern();
     }
-
+    
```

Comment:
Hey @cedarvinda 
Can you please remove extra space in line no 118.

Reply:
Thankyou @ronak2ram , Yes sure removing added by mistake

Reply:
Removed

Reply:
@cedarvinda thanks for the update. now, PR looks good.